nixos/modules/services/web-servers/varnish/default.nix

```diff
@@ -7,32 +7,48 @@ with lib;
 {
   options = {
     services.varnish = {
-      enable = mkOption {
-        default = false;
-        description = "
-          Enable the Varnish Server.
-        ";
-      };
+      enable = mkEnableOption "Varnish Server";
 
       http_address = mkOption {
```

Comment:
You could use `mkRenamedOptionModule` just in case. But just changing it would also be fine if it was broken anyway (i.e. also in 17.03).